app/views/admin/users/new.html.slim

```diff
@@ -5,23 +5,23 @@ div
     .form-inputs
       = f.label :first_name, "First Name"
       = f.text_field :first_name
-      
+
```

Comment:
Vim automatically strips whitespace for me on save to avoid introducing noise in PRs.